api/config/config_staging.json

```diff
@@ -2,6 +2,7 @@
   "firecloud": {
     "billingAccountId": "014D91-FCB792-33D2C0",
     "billingProjectPrefix": "aou-rw-staging-",
+    "clusterMaxAgeDays": 5,
```

Comment:
I'd actually say staging can probably be pretty small. Maybe even less than 5 days, just because its being used for QA and should be relatively one-off, correct?

Reply:
I think it should be the same as in test. Feel free to argue for that value being lower/higher.